webapp/pages/profile/_id/_slug.vue

```diff
@@ -204,7 +204,7 @@
               :key="post.id"
               :post="post"
               :width="{ base: '100%', md: '100%', xl: '50%' }"
-              @deletePost="Post.splice(index, 1)"
+              @removePostFromList="activePosts.splice(index, 1)"
             />
           </template>
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-06T15:35:01Z" title="Thursday, June 6th 2019, 5:35:01 pm +02:00">Jun 6, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -266,6 +267,35 @@ export default {
       voted: false,
       page: 1,
       pageSize: 6,
+      modalsData: {
+        delete: {
+          titel: () => {
```


```suggestion
          title: () => {
```

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-06-07T12:58:00Z" title="Friday, June 7th 2019, 2:58:00 pm +02:00">Jun 7, 2019</time>_

---


I have corrected this in my code …